app/internal/page/app.go

```diff
@@ -175,16 +168,29 @@ func App(
 
 	handleRoot(paths.Start, None,
 		Guidance(tmpls.Get("start.gohtml"), paths.Auth, nil))
-
 	handleRoot(paths.Dashboard, RequireSession,
 		Dashboard(tmpls.Get("dashboard.gohtml"), lpaStore))
 
+	handleRoot(paths.CertificateProviderStart, None,
+		CertificateProviderStart(tmpls.Get("certificate_provider_start.gohtml"), lpaStore))
+	handleRoot(paths.CertificateProviderLogin, None,
+		CertificateProviderLogin(logger, oneLoginClient, sessionStore, random.String))
+	handleRoot(paths.CertificateProviderLoginCallback, None,
+		CertificateProviderLoginCallback(tmpls.Get("identity_with_one_login_callback.gohtml"), oneLoginClient, sessionStore, lpaStore))
+	handleRoot(paths.CertificateProviderYourDetails, RequireSession|RequireCertificateProvider,
+		Guidance(tmpls.Get("certificate_provider_your_details.gohtml"), "", lpaStore))
+
 	lpaMux := http.NewServeMux()
 
 	rootMux.Handle("/lpa/", routeToLpa(lpaMux))
 
 	handleLpa := makeHandle(lpaMux, logger, sessionStore, localizer, lang, rumConfig, staticHash, paths, RequireSession)
 
+	handleLpa("/testing-certificate-provider-start", None, func(appData AppData, w http.ResponseWriter, r *http.Request) error {
```

Comment:
this is just temporary so we have something to go to before getting to sending the email